svelte/components/Nft/NftMint.svelte

```diff
@@ -300,20 +394,41 @@
               </div>
 
               <div class="section">
-                <div class="titre">NFT file</div>
+                <div class="titre">NFT image file</div>
                 <div class="box-file">
-                  {#if image}
+                  {#if image && !noCover}
                     <div class="media media-photo">
                       <img src={image} alt="nft" />
                       <span class="kre-delete-file" on:click={resetFileImg} on:keydown={resetFileImg}
                         ><i class="fa fa-trash" aria-hidden="true" /></span
                       >
                     </div>
                   {:else}
-                    <input type="file" id="file" name="file" bind:files on:change={fileload} />
+                    <div class="kre-flex kre-baseline">
+                      <input
+                        type="file"
+                        id="file"
+                        name="file"
+                        bind:files
+                        on:change={fileload}
+                        accept={acceptedImgTypes}
+                        disabled={noCover}
+                      />
+                      {#if inputMediaType === "audio"}
+                        <div class="kre-no-cover">
+                          <input id="kre-no-cover" type="checkbox" bind:checked={noCover} />
+                          <label for="kre-no-cover">no cover</label>
```

Comment:
rather than a "no cover" checkbox, just display the default cover that can be change 
(as default values in text fields)

audio file should come first , cover second (and optional)

detail what format are possible ?